### ISSUE TITLE
Poetry could fail with Python on Win10

### ISSUE THREAD

**Issue by ivanshilo** [rgerum/cameratransform#22]
I have tried to install newest version of cameratransform that now requires poetry. (Documentation update anyone?)
It turns out that one must install python with normal installation under Win10 that uses user's local folder for installation like
'%userprofile%\AppData\Local\Programs\Python\Python311\'
If you install python with MS Store app, it will be sandboxed into 'C:\Users\shilo_i\AppData\Local\Packages\Python...' and this is not going smooth with poetry.

**Comment by rgerum** [comment]
Hmm but that seems to be a problem that poetry should resolve, no? Its not intrinsically a problem with cameratransform. In theory packages build with poetry should be installable with pip (at least as far as I understood).

**Comment by ivanshilo** [comment]
> Hmm but that seems to be a problem that poetry should resolve, no? Its not intrinsically a problem with cameratransform. In theory packages build with poetry should be installable with pip (at least as far as I understood).

That's correct. I put it here as a hint for others instead of telling them: "go find a known issue on poetry github" (Which I didn't find there)
